tools/converter/reports_converter.cpp

```diff
@@ -0,0 +1,126 @@
+#include <cstring>
+#include <fstream>
+#include <iostream>
+#include <map>
+#include <vector>
+
+#ifdef SONATA_REPORT_HAVE_MPI
+#include <mpi.h>
+#endif
+
+#include "binary_reader/binary_reader.h"
+#include <bbp/sonata/reports.h>
+#include <utils/logger.h>
+
+using namespace bbp::binary_reader;
+
+static void show_usage(std::string name) {
+    std::cerr << "Usage: " << name << " <report_filename> <report_type> [population_name]\n"
+              << "Options:\n"
+              << "\t-h,--help\t\tShow this help message\n"
+              << "\treport_filename\t\tName of the report to convert\n"
+              << "\treport_type\t\t'--soma' or '--compartment' required\n"
+              << "\tpopulation_name\t\tName of the report population (Default 'All')\n"
+              << "Examples:\n"
+              << "\t " << name << " soma.bbp --soma PopulationA\n"
+              << "\t " << name << " compartment.bbp --compartment PopulationB" << std::endl;
+}
+
+bool help(int argc, char* argv[]) {
+    for (int i = 1; i < argc; ++i) {
+        const std::string arg = argv[i];
+        if (arg == "-h" || arg == "--help") {
+            return true;
+        }
+    }
+    return false;
+}
+
+int main(int argc, char* argv[]) {
+#ifdef SONATA_REPORT_HAVE_MPI
+    MPI_Init(nullptr, nullptr);
+#endif
+    if ((argc != 3 && argc != 4) || help(argc, argv)) {
+        show_usage(argv[0]);
```

Comment:
nitpick?
```suggestion
        show_usage(argv);
```

Reply:
This is passing the name of the executable to the show_usage(std::string) function, so it should be argv[0]